### PR TITLE
mac RC1 updates and kill RU sentences and dead wood

### PR DESCRIPTION
**Pull request by Rick-Anderson**
working with PU on final wording.

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@scottaddie I'll have some minor tweaks but this pretty close.
@Erikre can you test all these on your Mac. I can send you the HTML if you like.
@rachelappel  when we get the tree rat, can you handle merger conflicts or should I get you some help?

**Comment by Rick-Anderson** [comment]
@DamianEdwards  the .NET Core support for Mac and Linux is still very new and not ready for production workloads.

**Comment by Rick-Anderson** [comment]
@rachelappel per Wades new policy can you merge this? All checks have passed
Anything @Erikre and others find I'll fix. These doc's were totally broken and now they're in good shape.

**Comment by Rick-Anderson** [comment]
@davidsan ready for review

**Comment by scottaddie** [comment]
@Rick-Anderson Thanks for the update! A couple quick questions for you:

1. Should the hyphen be removed from "client-tools"?
2. Should the code block type for the project.json "commands" snippet be changed from "c#" to "json"?

**Comment by Rick-Anderson** [comment]
@scottaddie  fixed, thanks

**Comment by davidsan** [comment]
The Person.cs file generated by yo has lost its indentation in the `yeoman.rst` file.
Otherwise, :+1:

**Comment by Rick-Anderson** [comment]
@rachelappel  looks like we have a couple tree rats

**Comment by Rick-Anderson** [comment]
@rachelappel  I did a commit on a bug fix for my MVC tutorial in this. Hope that doesn't mess things up.

File: aspnet/client-side/yeoman.rst [hunk 1]
```diff
@@ -1,95 +1,92 @@
...
-By `Noel Rice`_, `Scott Addie`_
+By `Scott Addie`_, `Rick Anderson`_ and `Noel Rice`_
 
-`Yeoman <http://yeoman.io/>`_ generates complete, running projects for a given set of client-tools. You may be overwhelmed by "feature-shock" for the latest glut of client-tools (NPM, Gulp, Angular, etc.) or have simply run out of time to work through each feature set, much less the details, of every client tool. Yeoman generates everything you need to get over your first hurdle, the "hello world" that demonstrates that a group of technologies function together. 
+`Yeoman <http://yeoman.io/>`_ generates complete projects for a given set of client tools. Yeoman is an open-source tool that works like a Visual Studio project template. The Yeoman command line tool `yo <https://github.com/yeoman/yo>`__ works alongside a Yeoman generator. Generators define the technologies that go into a project. 
 
-Yeoman is an open-source tool that works like a Visual Studio project template, but targets a wide developer audience that ranges from Ruby, browser extensions, PhoneGap, Facebook React, jQuery Mobile, and Microsoft technologies like ASP.NET.
+.. contents:: Sections:
+ :local:
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Please use two spaces for indented text

File: aspnet/client-side/yeoman.rst [hunk 2]
```diff
@@ -1,95 +1,92 @@
...
-- **Class Library**: A DNX-based class library.
-- **Unit Test project**: A simple `XUnit <http://xunit.github.io/>`_ test project. 
-
-This walkthrough demonstrates how to use Yeoman to generate an ASP.NET 5 web application. 
+- Install `Node.js <https://nodejs.org/en/>`__. The installer includes `Node.js <https://nodejs.org/en/>`__ and `npm <https://www.npmjs.com/>`__.
  
-1. Follow the instructions on http://yeoman.io/learning/ to install **yo** and other required tools. 
+- Follow the instructions on http://yeoman.io/learning/ to install `yo <https://github.com/yeoman/yo>`__, bower, grunt, and gulp.
+
+	- ``npm install -g yo bower grunt-cli gulp``
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Please use two spaces instead of tabs

File: aspnet/client-side/yeoman.rst [hunk 3]
```diff
@@ -1,95 +1,92 @@
...
+- From the command line, install the ASP.NET generator: 
 
 .. code-block:: console
 
  npm install -g generator-aspnet
+ 
+.. note:: If you get a permission error, run the command under ``sudo`` as described above.
 
-.. note:: The ``–g`` flag installs the generator globally, so that it can be used from any path on your system.
+- 	- The ``–g`` flag installs the generator globally, so that it can be used from any path.
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Remove double hyphen and use spaces instead of tabs

File: aspnet/client-side/yeoman.rst [hunk 4]
```diff
@@ -1,95 +1,92 @@
...
+ 
+- The generator displays a menu. Arrow down to the **Web Application** project and tap **Enter**:
 
 .. image:: yeoman/_static/yeoman-yo-aspnet.png
 
-7. Provide an application name of "MyWebApp" and press ``Enter``:
+- Use "MyWebApp" for the app name and then tap **Enter** :
 
-.. image:: yeoman/_static/yeoman-yo-aspnet-appname.png
+	.. image:: yeoman/_static/yeoman-yo-aspnet-appname.png
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Uses two spaces instead of tabs

File: aspnet/client-side/yeoman.rst [hunk 5]
```diff
@@ -1,95 +1,92 @@
...
 
-Yeoman will scaffold the project and its supporting files. Suggested next steps are also provided in the form of commands. 
+	Yeoman will scaffold the project and its supporting files. Suggested next steps are also provided in the form of commands. 
 
-.. image:: yeoman/_static/yeoman-yo-aspnet-created.png
+	.. image:: yeoman/_static/yeoman-yo-aspnet-created.png
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Use two spaces instead of tabs

File: aspnet/client-side/yeoman.rst [hunk 6]
```diff
@@ -1,95 +1,92 @@
...
+
+Create an ASP.NET app
+-------------------------
  
-3. Create the new directory in which the project will be generated:
+- Create a directory for the project
 
 .. code-block:: console
 
  mkdir C:\MyYo
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Can you do a pass on this doc to change all the single space indentations to be two spaces?

File: aspnet/client-side/yeoman.rst [hunk 7]
```diff
@@ -1,95 +1,92 @@
...
+- **Web Application Basic**: A simpler MVC web application lacking membership and authorization, among other advanced features which are found only in the **Web Application** generator. 
+- **Web API Application**: A Web API built with MVC. 
+- **Nancy ASP.NET Application**: A lightweight HTTP service with one module built using `Nancy <http://nancyfx.org/>`_.
+- **Class Library**: A DNX-based class library.
+- **Unit Test project**: A simple `XUnit <http://xunit.github.io/>`_ test project. 
+
+Setting Grunt as the task runner
+---------------------------------
+
+:doc:`Grunt </client-side/using-grunt>`
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Remove these references and instead inline them where Gulp and Grunt are first mentioned.

File: aspnet/client-side/yeoman.rst [hunk 8]
```diff
@@ -109,14 +106,13 @@ You can load your generated ASP.NET 5 web project directly into Visual Studio 20
 3.	When all the dependencies are restored, press **F5** to run the project. The default home page displays in the browser.
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Replace tab after the `3.` with a space

File: aspnet/client-side/yeoman.rst [hunk 9]
```diff
@@ -144,16 +140,16 @@ You can prepare and host your web application using commands **dnu** (Microsoft
...
+.. code-block:: json
+ :linenos:
+ :emphasize-lines: 2
 
-  "commands": {
-    "web": "Microsoft.AspNet.Server.Kestrel",
-    "ef": "EntityFramework.Commands"
-  },
+ "commands": {
+ "web": "Microsoft.AspNet.Server.Kestrel",
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Fix JSON formatting to indent correctly.

File: aspnet/client-side/yeoman.rst [hunk 10]
```diff
@@ -175,21 +171,24 @@ The result is a file named Person.cs with a class named ``Person``:
...
+ using System.Collections.Generic;
+ using System.Linq;
+ using System.Threading.Tasks;
  
-Summary
--------
-Yeoman generates complete running projects for a wide range of technology combinations. The generated files can be loaded into Visual Studio. Task Runner Explorer and other Visual Studio tooling help configure and automate unfamiliar technologies in a familiar environment. 
+ namespace MyNamespace
+ {
+ public class Person
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Fix C# formatting to indent correctly.

File: aspnet/client-side/yeoman.rst [hunk 11]
```diff
@@ -175,21 +171,24 @@ The result is a file named Person.cs with a class named ``Person``:
...
+ public Person()
+ {
+ }
+ }
+ }
+ 
+Related Resources
+-----------------
+
+- `HttpPlatformHandler, Kestrel and WebListener <https://github.com/aspnet/Home/wiki/Servers>`__
```
Comment (danroth27) [inline aspnet/client-side/yeoman.rst]: Replace GitHub wiki link with official doc link:
```
:doc:`Servers (HttpPlatformHandler, Kestrel and WebListener) </fundamentals/servers>`
```

File: aspnet/getting-started/choosing-the-right-dotnet.rst [hunk 12]
```diff
@@ -1,37 +1,65 @@
 Choosing the Right .NET For You on the Server
 =============================================
 
-By `Daniel Roth`_
+By `Daniel Roth`_ and `Rick Anderson`_
 
-ASP.NET 5 is based on the :doc:`.NET Execution Environment (DNX) </dnx/overview>`, which supports running cross-platform on Windows, Mac and Linux. When selecting a DNX to use you also have a choice of .NET flavors to pick from: .NET Framework (CLR), :doc:`.NET Core </conceptual-overview/dotnetcore>` (CoreCLR) or `Mono <http://mono-project.com>`_. Which .NET flavor should you choose? Let's look at the pros and cons of each one.
+ASP.NET 5 is based on the :doc:`.NET Execution Environment (DNX) </dnx/overview>`, which supports running cross-platform on Windows, Mac and Linux. DNX come in three .NET versions: 
+
+- .NET Framework (Also known as the "*Full*" .NET Framework)
```
Comment (danroth27) [inline aspnet/getting-started/choosing-the-right-dotnet.rst]: I don't think we ever use the "Full .NET Framework" with a capital "F". I would remove the comment in the parens.

File: aspnet/getting-started/choosing-the-right-dotnet.rst [hunk 13]
```diff
@@ -1,37 +1,65 @@
...
 
-`Mono <http://mono-project.com>`_ is a port of the .NET Framework built primarily for non-Windows platforms. Mono is open source and cross-platform. It also shares a similar API factoring to the .NET Framework, so many existing managed libraries work on Mono today. Mono is not a supported platform by Microsoft. It is however a good proving ground for cross-platform development while cross-platform support in .NET Core matures.
+`Mono <http://mono-project.com>`_ is a port of the .NET Framework built primarily for non-Windows platforms. Mono is open source and cross-platform. It also shares a similar API factoring to the .NET Framework, so many existing managed libraries work on Mono today. Mono is not a supported platform by Microsoft. It's a good proving ground for cross-platform development while .NET Core matures.
+
+Related Resources
+-----------------
 
-Summary
--------
+- :doc:`/fundamentals/index`
```
Comment (danroth27) [inline aspnet/getting-started/choosing-the-right-dotnet.rst]: Neither of these are really related to this topic. You could add a link to the DNX docs.

File: aspnet/getting-started/installing-on-mac.rst [hunk 14]
```diff
@@ -1,21 +1,21 @@
...
 ================================
 
-By `Steve Smith`_, `Daniel Roth`_
+By `Daniel Roth`_, `Steve Smith`_, `Rick Anderson`_
 
-.. contents:: In this article
-  :local:
-  :depth: 1
+.. contents:: Sections
+ :local:
```
Comment (danroth27) [inline aspnet/getting-started/installing-on-mac.rst]: Two spaces

File: aspnet/getting-started/installing-on-mac.rst [hunk 15]
```diff
@@ -1,21 +1,21 @@
...
+ :depth: 1
 
 Install ASP.NET 5 with Visual Studio Code
 -----------------------------------------
 
 The easiest way to get started building applications with ASP.NET 5 is to install the latest version of Visual Studio Code.
 
-1. Install `Visual Studio Code <https://go.microsoft.com/fwlink/?LinkID=534107>`__
+#. `Update OS X for Visual Studio Code <https://code.visualstudio.com/docs/supporting/howtoupdate#_updating-on-os-x>`__
```
Comment (danroth27) [inline aspnet/getting-started/installing-on-mac.rst]: Most people on OS X won't have Visual Studio Code at all. I would leave the original text (although the fwlink should be to 534106 - this was fixed recently).

File: aspnet/getting-started/installing-on-mac.rst [hunk 16]
```diff
@@ -26,13 +26,19 @@ You can also install ASP.NET 5 from the command-line. There are a few steps invo
 Install the .NET Version Manager (DNVM)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Use the .NET Version Manager (DNVM) to install different versions of the .NET Execution Environment (DNX) on OS X.
+To install DNVM:
 
-To install DNVM run the following::
+#. Open a Terminal.
+#. Type ``cd ~/`` to go to your home folder.
+#. Enter ``touch .bash_profile`` to create a new bash profile.
```
Comment (danroth27) [inline aspnet/getting-started/installing-on-mac.rst]: This is only necessary if you don't already have a bash profile and the install script will tell you to do this if you need to. Let's just leave the curl command.

File: aspnet/getting-started/installing-on-mac.rst [hunk 17]
```diff
@@ -26,13 +26,19 @@ You can also install ASP.NET 5 from the command-line. There are a few steps invo
...
+#. Run the following ``curl`` command::
 
-  curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh && source ~/.dnx/dnvm/dnvm.sh
+	curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh && source ~/.dnx/dnvm/dnvm.sh
```
Comment (danroth27) [inline aspnet/getting-started/installing-on-mac.rst]: Looks like a tab snuck in here.

File: aspnet/getting-started/installing-on-mac.rst [hunk 18]
```diff
@@ -26,13 +26,19 @@ You can also install ASP.NET 5 from the command-line. There are a few steps invo
...
 
-Once this step is complete you should be able to run ``dnvm`` and see some help text.
+#. Run ``dnvm list`` to show the default DNX version
```
Comment (danroth27) [inline aspnet/getting-started/installing-on-mac.rst]: Generally at this point you haven't installed any DNXs yet (installing one is the next step), so you won't have a default DNX version.

File: aspnet/getting-started/installing-on-mac.rst [hunk 19]
```diff
@@ -41,24 +47,25 @@ The .NET Execution Environment (DNX) is used to build and run .NET projects. Use
 
 **To install DNX for .NET Core:**
 
-1. Use DNVM to install DNX for .NET Core::
+1. Use DNVM to install DNX for .NET Core:
 
-    dnvm upgrade -r coreclr
```
Comment (danroth27) [inline aspnet/getting-started/installing-on-mac.rst]: Use two spaces instead of tabs.

File: aspnet/getting-started/installing-on-mac.rst [hunk 20]
```diff
@@ -41,24 +47,25 @@ The .NET Execution Environment (DNX) is used to build and run .NET projects. Use
...
 
 **To install DNX for Mono:**
 
 1. Install `Mono <http://www.mono-project.com/docs/getting-started/install/mac/>`__ for OS X. Alternatively you can install Mono via `Homebrew <http://brew.sh/>`__.
 
-2. Use DNVM to install DNX for Mono::
+2. Use DNVM to install DNX for Mono:
 
-    dnvm upgrade -r mono
+	``dnvm upgrade -r mono``
```
Comment (danroth27) [inline aspnet/getting-started/installing-on-mac.rst]: Use two spaces instead of tabs

File: aspnet/getting-started/installing-on-mac.rst [hunk 21]
```diff
@@ -41,24 +47,25 @@ The .NET Execution Environment (DNX) is used to build and run .NET projects. Use
...
+1. Use DNVM to install DNX for .NET Core:
 
-    dnvm upgrade -r coreclr
+	``dnvm upgrade -r coreclr``
 
 **To install DNX for Mono:**
 
 1. Install `Mono <http://www.mono-project.com/docs/getting-started/install/mac/>`__ for OS X. Alternatively you can install Mono via `Homebrew <http://brew.sh/>`__.
 
-2. Use DNVM to install DNX for Mono::
```
Comment (danroth27) [inline aspnet/getting-started/installing-on-mac.rst]: The 2nd colon means that the following indented text is preformatted, so you don't have to add the double back ticks.

File: aspnet/getting-started/installing-on-mac.rst [hunk 22]
```diff
@@ -41,24 +47,25 @@ The .NET Execution Environment (DNX) is used to build and run .NET projects. Use
...
 
-.. note:: Restoring packages using DNX on Mono may fail with multiple canceled requests. You may be able to work around this issue by setting ``MONO_THREADS_PER_CPU`` to a larger number (ex. 2000).
+.. note:: Restoring packages using DNX on Mono may fail with multiple canceled requests. You may be able to work around this issue by setting ``MONO_THREADS_PER_CPU`` to a larger number (2000).
 
 Related Resources
 -----------------
 
 - :doc:`/tutorials/your-first-mac-aspnet`
 - :doc:`/fundamentals/index`
+- `HttpPlatformHandler, Kestrel and WebListener <https://github.com/aspnet/Home/wiki/Servers>`__
```
Comment (danroth27) [inline aspnet/getting-started/installing-on-mac.rst]: Unrelated - remove.

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 23]
```diff
@@ -1,60 +1,46 @@
 Your First ASP.NET 5 Application on a Mac
 =========================================
 
-By `Steve Smith`_, `Daniel Roth`_
+By `Daniel Roth`_, `Steve Smith`_, `Rick Anderson`_
 
-ASP.NET 5 is cross-platform and can be developed and run on Mac OS X as well as Linux and Windows. See how you can quickly install, scaffold, run, debug, and deploy ASP.NET applications on a Mac.
+ASP.NET 5 is cross-platform; you can develop and run web apps on Mac OS X, Linux and Windows.
```
Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: How about something like: "Let's get started writing our first ASP.NET 5 application on a Mac"

Or: "This article will show you how to write your first ASP.NET 5 application on a Mac"

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 24]
```diff
@@ -1,60 +1,46 @@
...
+By `Daniel Roth`_, `Steve Smith`_, `Rick Anderson`_
 
-ASP.NET 5 is cross-platform and can be developed and run on Mac OS X as well as Linux and Windows. See how you can quickly install, scaffold, run, debug, and deploy ASP.NET applications on a Mac.
+ASP.NET 5 is cross-platform; you can develop and run web apps on Mac OS X, Linux and Windows. 
 
-.. contents:: In this article:
-  :local:
-  :depth: 1
+.. contents:: Sections:
+ :local:
```
Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: Two spaces

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 25]
```diff
@@ -65,10 +51,10 @@ This empty project template simply displays "Hello World!". Open ``Startup.cs``
 If this is your first time using Visual Studio Code (or just *Code* for short), note that it provides a very streamlined, fast, clean interface for quickly working with files, while still providing tooling to make writing code extremely productive. 
 
 In the left navigation bar, there are four icons, representing four viewlets:
-  - Explore
-  - Search
-  - Git
-  - Debug
+ - Explore
```
Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: Remove all leading whitespace from the bullets.

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 26]
```diff
@@ -85,57 +71,53 @@ Finally, Code's editor has a ton of great features. You should note right away t
 Running Locally Using Kestrel
 -----------------------------
 
-The sample we're using is configured to use Kestrel as its web server. You can see it configured in the ``project.json`` file, where it is specified as a dependency and as a command.
+The sample is configured to use `Kestrel <https://docs.asp.net/en/latest/fundamentals/servers.html#kestrel>`__ for the web server. You can see it configured in the ``project.json`` file, where it is specified as a dependency and as a :doc:`command </dnx/commands>`.
```
Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: Replace the hard coded link with a `:ref:`.

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 27]
```diff
@@ -85,57 +71,53 @@ Finally, Code's editor has a ton of great features. You should note right away t
...
-    },
-    // more deleted
-  }
-
-Run the ``dnx kestrel`` command to launch the web application locally:
-
-.. image:: your-first-mac-aspnet/_static/dnx-kestrel.png
-
-Navigate to ``localhost:5000`` and you should see:
+ :linenos:
```
Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: Fix up JSON formatting with correct indentation.

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 28]
```diff
@@ -145,51 +127,43 @@ Once you've developed your application, you can easily use the Git integration b
...
 
-First, if you haven't already done so, initialize Git in the folder you're working in. Simply click on the Git viewlet and click the ``Initialize Git repository`` button.
+Initialize Git in the folder you're working in. Tap on the Git viewlet and click the ``Initialize Git repository`` button.
 
 .. image:: your-first-mac-aspnet/_static/vscode-git-commit.png
 
-Add a commit message as shown in the image above, and press enter or click the checkmark icon to commit the staged files. Now Git is tracking changes, so if you make an update to a file, the Git viewlet will display how many files have changed since your last commit.
+Add a commit message and tap enter or tap the checkmark icon to commit the staged files. 
+
+.. image:: your-first-mac-aspnet/_static/init_commit.PNG
```
Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: File names should be all lowercase.

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 29]
```diff
@@ -145,51 +127,43 @@ Once you've developed your application, you can easily use the Git integration b
...
 
 .. image:: your-first-mac-aspnet/_static/azure-portal.png
 
-In a Terminal window, add a remote named ``azure`` with the Git URL you noted previously, and then perform ``git push azure master`` to deploy. You should see output similar to the following:
-
-.. image:: your-first-mac-aspnet/_static/git-push-azure-master.png
+- In a Terminal window, add a remote named ``azure`` with the Git URL you noted previously.
 
-Now you can browse to your Web App and you should see your newly deployed application.
+	- ``git remote add azure https://Rick-Anderson@rickmac.scm.azurewebsites.net:443/rickmac.git``
```
Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: Use two spaces instead of tabs

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 30]
```diff
@@ -145,51 +127,43 @@ Once you've developed your application, you can easily use the Git integration b
...
+	
+- Push to master.
 
-.. image:: your-first-mac-aspnet/_static/azure-hello-world.png
+	- ``git push azure master`` to deploy.
```
Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: Use two spaces instead of tabs.

File: aspnet/tutorials/your-first-mac-aspnet.rst [hunk 31]
```diff
@@ -197,4 +171,5 @@ Additional Resources
 - `Visual Studio Code <https://code.visualstudio.com>`__
 - :doc:`/client-side/yeoman`
 - :doc:`ASP.NET Fundamentals </fundamentals/index>`
+- `HttpPlatformHandler, Kestrel and WebListener <https://github.com/aspnet/Home/wiki/Servers>`__
```
Comment (danroth27) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: Unrelated - remove.

**Comment by danroth27** [comment]
:watch:

File: aspnet/getting-started/choosing-the-right-dotnet.rst (hunk 12 above)
Comment (Rick-Anderson) [inline aspnet/getting-started/choosing-the-right-dotnet.rst]: So do you want it .NET Framework (Also known as the "*full*" .NET Framework) 
or do you want me to remove  (Also known as the "*full*" .NET Framework) 
There is a ginormous ambiguity in the name .NET Framework, and full disambiguate it. No so for you and the team, but anyone new to .Net is confused. Even MS blogs call it the full .net framework. 

You can see in there how depending on the application type you’d want to create (Client apps or Web applications and Services) you now have two choices:

- You can use the full .NET Framework that’s evolving and now is branded as .NET Framework 4.6

- You can use the smaller and scenario-specialized new framework called .NET Core 5.

A different point of view, more “framework-based” of the same technologies is this othe
see http://blogs.msdn.com/b/cesardelatorre/archive/2014/11/18/what-is-net-core-5-and-asp-net-5-within-net-2015-preview.aspx

Comment (danroth27) [inline aspnet/getting-started/choosing-the-right-dotnet.rst]: I want to just remove the comment. The official terms are .NET Framework and .NET Core.

File: aspnet/client-side/yeoman.rst (hunk 7 above)
Comment (Rick-Anderson) [inline aspnet/client-side/yeoman.rst]: @danroth27 writes "Remove these top references to grunt and gulp - they look out of place."
LOL, they were the first thing in the article, I moved them to the end to keep the previous author happy but you're right, they don't belong here. Gone now.

File: aspnet/tutorials/your-first-mac-aspnet.rst (hunk 23 above)
Comment (Rick-Anderson) [inline aspnet/tutorials/your-first-mac-aspnet.rst]: You are a good writer :--) (just watch the ro sentences).

**Comment by Rick-Anderson** [comment]
@danroth27  - should be ready. 5 hours after commit the build is still running.

**Comment by danroth27** [comment]
I merged this in (8ed92fd3b90417deb1829f7ca5ff66e32092a4a7), but I reverted the changes to the Choosing the Right .NET article. Significant changes to that article really need to be reviewed with the .NET team. Really this content should live in the .NET docs. Let work with the .NET team on this.

**Pull request closed without merging**